docs/websub.md

```diff
@@ -8,6 +8,14 @@ In MOSIP WebSub is used to share data with services and partners.  [Kafka](https
 
 ![](_images/websub.png)
 
+The relationship of Websub with other services is explained here.NOTE: The numbers do not signify sequence of operations or control flow
```

Comment:
Websub to be written as "WebSub". Please maintain consistency as per the terms used in docs.mosip.io

Reply:
done